src/c/luv_c_type_descriptions.ml

```diff
@@ -126,7 +126,9 @@ struct
     module Option =
     struct
       let block_signal = constant "UV_LOOP_BLOCK_SIGNAL" int
-      let sigprof = constant "SIGPROF" int
+      let sigprof = match Sys.win32 with 
+      | false -> constant "SIGPROF" int
```

Comment:
I'm not sure the best way to handle this, but `SIGPROF` isn't defined on Windows.